Gems/Atom/RHI/Code/Include/Atom/RHI/BufferView.h

```diff
@@ -25,6 +25,8 @@ namespace AZ
             AZ_RTTI(BufferView, "{3012F770-1DD7-4CEC-A5D0-E2FC807548C1}", ResourceView);
             virtual ~BufferView() = default;
 
+            static constexpr uint32_t InvalidBindlessIndex = 0xFFFFFFFF;
```

Comment:
Maybe easier to move this variable to Limits.h so we dont have multiple definitions

Reply:
It probably does make more sense to merge these two into one common place. There's not really a section in Limits for this kind of thing, but one could be added. We could also add it to the common ResourceView. Holding off on this for now since it's a minor change and I'd rather not re-run AR for it.